common/reviews/api/rush-lib.api.md

```diff
@@ -202,7 +232,13 @@ export interface ICredentialCacheOptions {
     supportEditing: boolean;
 }
 
-// @beta
+// @beta (undocumented)
+export interface IEnvironmentConfigurationInitializeOptions {
+    // (undocumented)
+    doNotNormalizePaths?: boolean;
+}
+
+// @public
 export interface IExperimentsJson {
```

Comment:
I don't think this should be considered a stable API, as things will disappear from it as we graduate experiments.

Reply:
Fixed